packages/contracts/deploy/env/00-env-check.ts

```diff
@@ -0,0 +1,53 @@
+import {
```

Comment:
🚩: should we worry about the fact that env/, new/, update/ don't have numbers as prefix?

Reply:
The files in `new/` and `update/` are independent deploy scripts (we either deploy freshly with `new` or update the protocol with `update/to_vX.Y.Z`. 
We `cd` into the path and only then the numbers become important and define the execution order of the scripts.

Reply:
in that case, we need to add a checklist item to both of our deploy and update checklist that the first file inside `/new`
and `update/to-xyz` have to has `env` dependency set properly, right?

Reply:
You can set dependencies as `func.dependencies` in each of the deploy scripts. 

What I've done:

- Tagged the 00-env-check with the tag `'Env'`
- Added `'Env'` as a dependency to the first deploy func in each folder

Thus, when running the script, the env check has to run first. 

We can make 2 improvements:

1. As Sarkawt points out: make this clear in the checklist
2. I don't like the use of string tags. We should have a simple const object that contains the tags:

```ts
const TAGS = {
  ENV: 'Env',
  MANAGEMENT_DAO: 'ManagementDao'
  // ... etc
} as const
```

Reply:
I think by definging them as `func.dependencies = ['Env'];`,  we make sure that these are run first. Can you confirm @jordaniza ?